adafruit_midi/midi_message.py

```diff
@@ -121,6 +121,24 @@ class MIDIMessage:
     # order is more specific masks first
     _statusandmask_to_class = []
 
+    def __init__(self, *, channel=None):
+        ###  TODO - can i kwargs this?????
```

Comment:
What do you mean by this?

Reply:
I was just pondering if I should/could iterate over the actual args passed and not set the value if it wasn't passed in, e.g. SystemExclusive does not set the value when it calls it. I don't think it's an important issue.

BTW, unrelated, I just threw in some defaults on velocity as it's useful on command line and I think it's 3 bytes per mpy.

(I thought Travis highlighted all the TODO left in the file but maybe I imagined that.)